packages/forms-web-app/src/views/error/not-found.njk

```diff
@@ -1,11 +1,31 @@
 {% extends "layouts/main.njk" %}
 
+{% block pageTitle %}
+  Page not found- Appeal a householder planning decision - GOV.UK
```

Comment:
I hate the lack of space after `found` and `-`.

But that's as per the ticket, so all good 👍